diff --git a/dist/rollup.js b/dist/rollup.js
index 65df1bfcedaf873043fd82492afbb7a2926654a5..6016815127beae690a5471d2162dce68ebc8f2e2 100644
--- a/dist/rollup.js
+++ b/dist/rollup.js
@@ -6197,7 +6197,9 @@ export { ${[...exportedNames].join(", ")} };
               }
             },
             treeshake: {
-              moduleSideEffects: "no-external",
+              moduleSideEffects: options.treeshake?.moduleSideEffects === undefined
+                ? "no-external"
+                : options.treeshake.moduleSideEffects,
               propertyReadSideEffects: true,
               unknownGlobalSideEffects: false
             }
@@ -6307,6 +6309,7 @@ export { ${[...exportedNames].join(", ")} };
         outputOptions: transformPlugin.outputOptions,
         renderChunk: transformPlugin.renderChunk,
         options(options2) {
+          options2.treeshake = options.treeshake;
           let { input = [] } = options2;
           if (!Array.isArray(input)) {
             input = typeof input === "string" ? [input] : Object.values(input);
@@ -6778,6 +6781,7 @@ var getRollupConfig = async (options) => {
         json(),
         ignoreFiles,
         dtsPlugin.default({
+          treeshake: dtsOptions.treeshake,
           tsconfig: options.tsconfig,
           compilerOptions: {
             ...compilerOptions,
